### DISCUSSION TITLE
ICRS Grid as an extra?

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2976] [Ideas]
ICRS grid is currently appearing as an extra in the configuration panel. Shouldn't it appear as a markings in the view panel?

**Comment by gzotti** [comment]
One configures the visibility of the button, the other configures the visibility of the grid. Probably we could rename ICRS to J2000. @alex-w ?

**Reply by alex-w** [reply]
No need to rename it

**Comment by diegorodriguezperegrin** [comment]
Under Markings, there are already "Eq. grid J2000" (labelled as ICRS in the lower tool bar), "Eq, grid of date" and "Fixed eq. grid".

Maybe it's not necessary to distinguish three types of equatorial markings, since in fact the reference frame will be that of the catalog in use, All modern catalogs comply with ICRS, so they refer to the solar system barycenter and are oriented as "Equatorial" but not referred to any equinox, if I am not wrong.

- Hipparcos (I/239) or Tycho-2 (I/259) catalogs use HCRF, epoch J1991.25. 
- NOMAD is a simple merge with UCAC2 (I/289), and USNO-B1.0 (I/284), epoch J2000.0. 
- Gaia DR3 (I/350), epoch J2016.0. 

Every new catalog showing increased astrometric, photometric, etc. data.

**Reply by gzotti** [reply]
epoch != equinox...

**Reply by diegorodriguezperegrin** [reply]
Oh, no. Both are different. Epoch (of a given date) refers to the actual measured coordinates of a star taking into account precession, nutation and proper motion. Equinox in the other hand refers to a reference frame defined by the positions of the ecliptic and the celestial equator as of the date/epoch on which the position of the star is being specified. Today, equinox dates are not used (except for civilian life) since they are difficult to precise (Earth orientation changes by the hour) and ICRS totally neglects them. 

To summarize:  Epoch= Date on which the object was at the quoted RA and Dec. Sometimes the epoch is the actual date when the object's position was measured, sometimes it's a prediction for some standard date (e.g. 2000), based on the known proper motion, and a measured position at some other time. Equinox determines the coordinate system used, usually J2000, but ICRF replaced it. In fact, their axes agree to within 0.02 arcseconds. It also supersedes older equinox-based systems like B1900 and B1950.

**Reply by gzotti** [reply]
"!=" means "not equal", so we at least agree on that.

In the context of this program, ICRF==Eq.J2000.0. I understand star catalog epochs as positions w.r.t. proper motion in J2000 frame.  Switch on "fixed eq. grid" to see what it is.  Our planet positions finally coincide with JPL Horizon to within fractions of arcseconds, see Appendix F in the User Guide. It should be enough to locate objects and guide telescopes. If you want to be more accurate, feel free to improve with working code.

**Reply by diegorodriguezperegrin** [reply]
Oh, yes. I see the fixed grid to be a non rotating grid, a projection of the Earth grid. Nice.

By the way, why using "!=" when you can use "≠"? Ok, Stellarium keeps impressing me everday. 

Thank you for your patience!
